docs/configuration/alertmanager-integration/grafana-alert-manager.rst

```diff
@@ -1,31 +1,108 @@
 Grafana AlertManager
 ****************************************
 
-If you use Grafana alerts, you're using a `special instance of AlertManager embedded inside Grafana <https://grafana.com/docs/grafana/latest/alerting/fundamentals/alertmanager/>`_.
+Using Grafana alerts involves a `special instance of AlertManager embedded within Grafana <https://grafana.com/docs/grafana/latest/alerting/fundamentals/alertmanager/>`_.
 
-You will need to configure to send Grafana alerts to Robusta.
+You will need to configure two integrations one to send alerts to Robusta and another to let Robusta query metrics and create silences.
```

Comment:
Missing colon:

You will need to configure two integrations`:` one to send alerts to Robusta and another to let Robusta query metrics and create silences.